config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-
+  root 'home#index'
+  resources :posts
```

Comment:
We typically whitelist which routes/controller actions are accessible (of the 7) so that we don't have unused routes hanging around with no corresponding controller action.

For example, it looks like you probably won't be using the `posts#index` route/action.

You can whitelist these by doing: `resources :posts, only: [:new, :create, :show]`. I realize some more will get added in the future, but being explicit here is a good practice to get in the habit of.